openlibrary/templates/type/edition/view.html

```diff
@@ -1,9 +1,9 @@
 $def with (page)
 
 $ work = page.works and page.works[0] or None
-$ book_title = page.get('title', '') if page.title else work.title if work else ''
+$ book_title = page.get('title', '') if page.title else (work.title if work else '')
 $ work_title = work.title if work else page.get('title', '')
-$ book_subtitle = page.subtitle if page.get('subtitle') else work.subtitle if work and work.get('subtitle') else ''
+$ book_subtitle = page.subtitle if page.get('subtitle') else (work.subtitle if work and work.get('subtitle') else '')
```

Comment:
This seems to permit display of the odd combination of edition title with work subtitle, as seen at https://openlibrary.org/books/OL9873518M/Hijas_de_la_alegria?v=7